crypto/dh/dh_key.c

```diff
@@ -271,6 +271,7 @@ static int generate_key(DH *dh)
             if (dh->params.q == NULL) {
                 /* secret exponent length */
                 l = dh->length ? dh->length : BN_num_bits(dh->params.p) - 1;
+                /* TODO: should l be checked for validity here? */
```

Comment:
Probably 😀 

If the other branch of the _if_ above, there is an error check: `dh->length > BN_num_bits(dh->params.q)`.

Reply:
Hmmm, this TODO was supposed to stay in #13140...